ovc/client.go

```diff
@@ -137,19 +174,126 @@ func (c *Client) Do(req *http.Request) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
+	taskID := string(body)
 
 	c.logger.Debug("OVC call: " + req.URL.Path)
 	c.logger.Debug("OVC response status code: " + resp.Status)
-	c.logger.Debug("OVC response body: " + string(body))
+	c.logger.Debug("OVC response body: " + string(taskID))
 
 	switch {
 	case resp.StatusCode == 401:
 		return nil, ErrAuthentication
 	case resp.StatusCode > 202:
-		return body, errors.New(string(body))
+		log.Printf("[DEBUG][SMURTH] resp > 202 status %v", resp.Status)
+		log.Printf("[DEBUG][SMURTH] resp > 202 body %v", taskID)
+		log.Printf("[DEBUG][SMURTH] resp > 202 full resp %v", resp)
+
+		return body, errors.New(taskID)
+	}
+
+	// remove quotes from taskID if contains any
+	var replacer = strings.NewReplacer("\"", "")
+	taskID = replacer.Replace(taskID)
+
+	// create request to get result of an async API call by job id
+	type TaskConfig struct {
```

Comment:
If you want you can use an anonymous struct as you define TaskConfig struct but never use it again anyway

```
taskJSON, err := json.Marshal(
    struct{
        TaskID string `json:"taskguid"`
    } {
         taskID
    },
 )
```